DigitalLearningSolutions.Data.Tests/DataServices/SelfAssessmentDataServiceTests/CandidateAssessmentsDataServiceTests.cs

```diff
@@ -0,0 +1,254 @@
+﻿namespace DigitalLearningSolutions.Data.Tests.DataServices.SelfAssessmentDataServiceTests
+{
+    using System;
+    using System.Transactions;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using FluentAssertions;
+    using NUnit.Framework;
+
+    public partial class SelfAssessmentDataServiceTests
+    {
+        [Test]
+        public void GetSelfAssessmentForCandidateById_should_return_a_self_assessment()
+        {
+            // When
+            var result = selfAssessmentDataService.GetSelfAssessmentForCandidateById(CandidateId, SelfAssessmentId);
+
+            // Then
+            var expectedSelfAssessment = SelfAssessmentHelper.CreateDefaultSelfAssessment(
+                SelfAssessmentId,
+                "Digital Capability Self Assessment",
+                "When thinking about your current role, for each of the following statements rate your current confidence level (Where are you now) and where your confidence leve ought to be to undertake your role successfully (Where do you need to be). Once you have submitted your ratings they will be used to recommend useful learning resources. We will also collect data anonymously to build up a picture of digital capability across the workforce to help with service design and learning provision.",

```

Comment:
Ridiculously long line here, could put it in a variable and concatenate bits together?